README.md

```diff
@@ -9,9 +9,9 @@ C library and executable tools for reading DICOM files.
 ### Building from source
 
 ```none
-./autogen.sh
-make
-make install
+meson setup -Dbuildtype=release builddir
```

Comment:
You can use `--buildtype release` fwiw. `setup` is the default, so `meson buildir --buildtype release` also works.

Reply:
Ah yeah, `--buildtype` is better.  `meson builddir` produces a warning on current versions of Meson, though.  Re `lib/x86_64-linux-gnu` (and likewise `lib64` on Fedora), isn't Meson doing the correct thing by using the directory that's in the distro's search path?

Reply:
> `setup` is the default, so `meson buildir --buildtype release` also works.

It will also raise a warning telling you it is "ambiguous and deprecated". :P 

Imagine if people do `meson build` and then expect it to create a build/ directory, but then Meson adds a "build" subcommand? That actually nearly happened, but we ended up using an inferior name: `meson compile`. Specifically due to fear of clashes.

> Re `lib/x86_64-linux-gnu` (and likewise `lib64` on Fedora), isn't Meson doing the correct thing by using the directory that's in the distro's search path?

Indeed, at least if you're going to install it to a distro search path.

If you're using a custom prefix somewhere in $HOME or something, then it probably doesn't matter either way (you really need rpath), but the debianlike path is longer and has more extraneous content in it, so some might say that's ugly...

Reply:
The Debian path causes problems with things like ffi within eg. python and ruby. Debian has `/usr/local/lib` as well as `/usr/local/lib/x86_64-linux-gnu` in the default search path, so plain `lib` is almost always better, I think.

Good to know about `meson setup`, I'll change the advice in the libvips readme, thanks!